jsBasics/homeWork5/task2.js

```diff
@@ -0,0 +1,6 @@
+function isPersonAdult(age){
+    let result = age >= 18;
```

Comment:
why let but not const?

Reply:
@sttaran I didn't find in our lesson that I can't use let in functions. But if points are deducted for this, then I corrected it.